governance/xc_admin/packages/xc_admin_cli/src/index.ts

```diff
@@ -924,4 +925,55 @@ multisigCommand("execute-add-and-delete", "Execute a roster change proposal")
     await vault.squad.executeTransaction(proposal);
   });
 
+multisigCommand(
+  "set-trusted-signer",
+  "Set a trusted signer for the Lazer program"
+)
+  .requiredOption(
+    "-s, --signer <pubkey>",
+    "public key of the trusted signer to add/update"
+  )
+  .requiredOption(
+    "-e, --expiry-time <seconds>",
+    "expiry time in seconds since Unix epoch. Set to 0 to remove the signer."
+  )
+  .action(async (options: any) => {
+    const vault = await loadVaultFromOptions(options);
+    const targetCluster: PythCluster = options.cluster;
+
+    const trustedSigner = new PublicKey(options.signer);
+    const expiryTime = new BN(options.expiryTime);
+
+    // Create the update instruction
+    const updateInstruction = new TransactionInstruction({
+      programId: LAZER_PROGRAM_ID,
+      keys: [
+        {
+          pubkey: await vault.getVaultAuthorityPDA(targetCluster),
+          isSigner: false,
+          isWritable: false,
+        },
+        {
+          pubkey: STORAGE_ID,
+          isSigner: false,
+          isWritable: true,
+        },
+      ],
+      data: Buffer.concat([
+        // Anchor discriminator for "update"
+        Buffer.from([219, 200, 88, 176, 158, 63, 253, 127]),
+        // Trusted signer pubkey
+        trustedSigner.toBuffer(),
+        // Expiry time (i64)
+        Buffer.from(expiryTime.toArray("le", 8)),
+      ]),
+    });
```

Comment:
Hmmm did you try using anchor program directly here?

Reply:
hmm thats a good call